packages/rest-specs/specs/azure/client-generator-core/access/main.tsp

```diff
@@ -0,0 +1,191 @@
+import "@typespec/http";
```

Comment:
The entire azure folder should not be included in this repo.

Reply:
Done